### PR TITLE
JoeYi Add template command

### LINKED ISSUE #73: As a User, I want to quickly access fields of each type of person
so that I can quickly fill in the details and not have to remember every field for each person, especially those with fields unique to them e.g. ratings for ta and prof, github for students and ta.

### PR DESCRIPTION
**Pull request by JJoeYi**
As the number of unique fields increase for each Person type (prof, student, ta), the more likely the users will forget what fields exist for which type of Person. This slows their productivity and makes our product more cumbersome.

Template command allows users to quickly access the fields for each class so that they can fill them in quickly.

Template command is executed using the "tt" command word.
e.g. `tt prof` or `tt student` or `tt ta`

Reason for `tt` rather than "template" was also to decrease the time it takes to get the fields.

Fixes #73

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/TemplateCommand.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        }
+        return new CommandResult(SHOWING_HELP_MESSAGE);
+    }
+
+    /**
+     * @param person The string representing the COMMAND_WORD of the Person chosen
+     * @return true if it is a valid COMMAND_WORD of a Person, false otherwise.
+     */
+    public static boolean isValidPerson(String person) {
+        if (person.equals(ProfCommand.COMMAND_WORD)) {
```
Comment (Jonaspng) [inline src/main/java/seedu/address/logic/commands/TemplateCommand.java:56]: Could technically use `||` to make this a one liner.

File: src/main/java/seedu/address/logic/commands/TemplateCommand.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+     * @param personChosen the command word of a Person.
+     */
+    public TemplateCommand(String personChosen) {
+        requireNonNull(personChosen);
+        this.personChosen = personChosen;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        if (personChosen.equals(ProfCommand.COMMAND_WORD)) {
```
Comment (Jonaspng) [inline src/main/java/seedu/address/logic/commands/TemplateCommand.java:37]: Should add a `requireNonNull(model)` here for defensive coding

**Review by Jonaspng — APPROVED** [review]
LGTM. But please look through my suggestions before merging

File: src/main/java/seedu/address/logic/commands/TemplateCommand.java (hunk 2 above)
Comment (JJoeYi) [reply src/main/java/seedu/address/logic/commands/TemplateCommand.java:37]: Thanks!

**Pull request merged**